website/sidebarsIntegrations.js

```diff
@@ -105,6 +105,7 @@ module.exports = {
                         "services/sonarr/index",
                         "services/tautulli/index",
                         "services/weblate/index",
+                        "services/immich/index",
```

Comment:
We try to keep this list alphabetically sorted.

Reply:
Sorry, makes sense!